msteams-platform/sbs-teams-conversation-bot.yml

```diff
@@ -70,16 +70,16 @@ items:
     1. Enter bot handle name in **Bot handle** field.
     1. Select your **Subscription** from the dropdown list.
     1. Select your **Resource group** from the dropdown list.
-
-        To create a new resource group, select **Create new**, enter resource name, select **OK**, and select required location from **New resource group location** dropdown list.
+    1. To create a new resource group, select **Create new**, enter resource name, select **OK**, and select required location from **New resource group location** dropdown list.
+       For more information see [Create resource group](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/manage-resource-groups-portal#create-resource-groups)
```

Comment:
Link to create resource group is not required here there is a link in the UI to create a resource group. Remove

Reply:
The link is removed as suggested.